packages/common/src/services/audius-api-client/AudiusAPIClient.ts

```diff
@@ -1544,6 +1557,43 @@ export class AudiusAPIClient {
     return response ? response.data : null
   }
 
+  async getPremiumContentSignatures({
+    userId,
+    trackMap
+  }: GetPremiumContentSignaturesArgs) {
+    if (!Object.keys(trackMap).length) return null
+
+    const encodedUserId = this._encodeOrThrow(userId)
+    this._assertInitialized()
+
+    // To avoid making a POST request and thereby introducing a new pattern in the DN,
+    // we build a param string that represents the info we need to verify nft collection ownership.
+    // The trackMap is a map of track ids -> token ids.
+    // If the nft collection is not ERC1155, then there are no token ids.
+    // We append the track ids and token ids as query params, making sure they're the same length
+    // so that DN knows which token ids belong to which track ids.
+    // Example:
+    // trackMap: { 1: [1, 2], 2: [], 3: [1]}
+    // query params: '?track_ids=1&token_ids=1-2&track_ids=2&token_ids=&track_ids=3&token_ids=1'
```

Comment:
(ik this is outside the scope of this pr, but i think we might want clean this up and make it order agnostic by using JSON in the query string or our own parseable thing like `?track_token_id_pairs=1:1,2&track_token_id_pairs=2:` or something? worth thinking about)